docs/static/breaking-changes-80.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[breaking-8.0]]
+=== Breaking changes in 8.0
+Here are the breaking changes for 8.0.
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+// tag::notable-breaking-changes[]
+[discrete]
+[[security-on-8.0]]
+===== Secure communication with {es} 
+{ls} must establish a secure connection using an {es} self-signed SSL certificate before it can transfer data to an on-premise {es} cluster. 
+{es} generates its own default self-signed Secure Sockets Layer (SSL) certificates at startup. 
+{ls} must have a copy of the {es} CA and the self-signed certicate to establish a secure connection. 
```

Comment:
Tricky:

Many on-prem ES's use full PKI (Public Key Infrastructure, with org-specific Certificate Authorities) and _not_ self-signed certs, and these notes are incorrect for those security-minded orgs.

The upstream change that we are reacting to is that _IFF_ an on-prem Elasticsearch cluster is started without certificates, the first node in the cluster bootstraps itself with self-signed certificate authority, which it uses to generate and sign certificates for other nodes in the cluster and anything that connects to it will need to be configured to trust certs signed by _that_ self-signed CA.

No matter how the cluster is configured, Logstash needs to have access to the relevant CA -- it's just that if the user _doesn't_ explicitly configure security on the Elasticsearch cluster, they need to know that the CA is self-signed and generated by the cluster (so they can know where to get it from).

```suggestion
{ls} must establish a Secure Sockets Layer (SSL) connection before it can transfer data to an on-premise {es} cluster. 
{ls} must have a copy of the {es} CA that signed the cluster's certificates.
When a new {es} cluster is started up _without_ dedicated certificates, it generates its own default self-signed Certificate Authority at startup.
```

Nitpick: the certs themselves aren't "SSL certificates" (that's like saying "a high-five hand" -- naming the thing after one of many possible uses)